modules/cache.js

```diff
@@ -0,0 +1,171 @@
+var async = require('async');
+var transactionTypes = require('../helpers/transactionTypes.js');
+var client, self, logger, cacheEnabled, cacheReady = true, errorCacheDisabled = 'Cache Unavailable';
```

Comment:
I thought about one `var` per line - we're following that standard because it's more readable. But for me is also fine to declare not assigned variables at once.